spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe "Posts", type: :request do
+ describe "GET #index" do
+   it "returns http success" do
+     user = create(:user) 
+     get user_posts_path(user)
+     expect(response).to have_http_status(:success)
+   end
+
+   it "renders the index template" do
+     user = create(:user) 
+     get user_posts_path(user)
+     expect(response).to render_template(:index)
```

Comment:
## optional 
* I strongly suggest you add tests for the show actions for the posts controller as this was missed by previous CR's i marked it as optional

Reply:
Ok 👍